superset-frontend/src/common/components/Tabs.tsx

```diff
@@ -16,27 +16,46 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import React from 'react';
 import { styled } from '@superset-ui/core';
 import { Tabs as AntdTabs } from 'src/common/components';
+import { css } from '@emotion/core';
+import Icon from '../../components/Icon';
```

Comment:
```suggestion
import Icon from 'src/components/Icon';
```
Using this webpack alias makes it easier to shuffle files around :)

Reply:
Done